cves/CVE-2012-3443.yml

```diff
@@ -264,7 +269,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: This is less of a mistake and more of an oversight, there is code to ensure
+    images displayed are legitimate, but the size and compression amount are not enforced
```

Comment:
You can also limit the size of images that can be uploaded and the amount of compression. It wasn't in this case, which feels like a design flaw to me.